apps/parse/lib/parse/trips.ex

```diff
@@ -60,4 +61,7 @@ defmodule Parse.Trips do
   end
 
   defp copy_if_not_blank(_), do: nil
+
+  defp parse_is_revenue_service("NONREV-" <> _), do: false
```

Comment:
@jfabi is this the best way we have to determine whether a trip is revenue or not?

Reply:
I thought the original plan was to add a `revenue: false` attribute to the TripDescriptor?

Reply:
I don't remember what the actual plan was, for some reason I hadn't realized that nonrevenue vehicles still had a TripDescriptor at all. But I don't think we want to be parsing the trip IDs, that's an implementation detail of RTR.